Equilobe.DailyReport.Models/Storage/Policy.cs

```diff
@@ -16,9 +17,18 @@ public class Policy
 
         public SourceControlOptions SourceControlOptions { get; set; }
 
-        public GeneratedProperties GeneratedProperties { get; set; }
-
         [XmlIgnore]
         public List<string> EmailCollection { get; set; }
+
+        public string RootPath { get; set; }
+
+        [XmlIgnore]
+        public string LogPath { get { return Path.Combine(RootPath, "Logs"); } }
+        [XmlIgnore]
+        public string LogArchivePath { get { return Path.Combine(RootPath, "LogArchive"); } }
+        [XmlIgnore]
+        public string ReportsPath { get { return Path.Combine(RootPath, "Reports"); } }
+        [XmlIgnore]
+        public string UnsentReportsPath { get { return Path.Combine(RootPath, "UnsentReports"); } }
```

Comment:
am dat merge din asta in celalalt la un moment dat ca trebuia sa lucrez cu baza de date